.github/workflows/build-for-pr.yml

```diff
@@ -123,6 +122,10 @@ jobs:
       - name: ci/build
         env:
           MM_WIN_INSTALLERS: 1
+          PFX_KEY: ${{ secrets.MM_DESKTOP_MSI_INSTALLER_PFX_KEY }}
```

Comment:
Will likely remove this after this PR is done being tested - I think we're okay to still just ship the ZIP for PR builds.